packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,244 @@
+
+To explicitly allow or deny registration to the user, supply a type property set to `com.okta.action.update`, together with a value property set to `{"registration": "ALLOW"}` or `{"registration": "DENY"`}. The default is to allow registration.
+
+Commands are applied in the order in which they appear in the array. Within a single command, attributes are updated in the order in which they appear in the `value` object.
+
+You can never use a commands to update the user's password, but you are allowed to set the values of attributes other than password which are designated sensitive in your Okta user schema. Note however that the values of those sensistive attributes, if  included as fields in the Self-Service Registration form, are not included in the `data.userProfile` object sent to your external service by Okta. See [data.userProfile](#data-userProfile) above.
+
+#### value
+
+The `value` object is the parameter to pass to the command.
+
+For `com.okta.user.profile.update` commands, `value` should be an object containing one or more name-value pairs for the attributes you wish to update, for example:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "middleName":"Danger",
+            "customerId":12345
+         }
+      }
+   ]
+}
+```
+
+> Note: The above example assumes that there is an attribute `customerId` defined in your Okta user schema (`middleName` is defined by default).
+
+The same result could also be accomplished by means of two separate `com.okta.user.profile.update` commands, as follows:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "middleName":"Danger"
+         }
+      },
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "customerId":12345
+         }
+      }
+   ]
+}
+```
+
+For `com.okta.action.update` commands, `value` should be an object containing the attribute `action` set to a value of either `ALLOW` or `DENY`, indicating whether the registration should be permitted or not, for example:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.action.update",
+         "value":{
+            "action":"DENY",
+         }
+      }
+   ]
+}
+```
+
+Registrations are allowed by default, so setting a value of `ALLOW` for the `action` field is valid but superfluous. 
+
+### error
+
+See [error](/use_cases/inline_hooks/) for general information on the structure to use for the `error` object.
+
+In the case of the Registration Inline Hook, the `error` object provides a way of causing an error message to be displayed to the end user who is trying to register. If you're using the Okta Sign-In Widget for Self-Service Registration, and have not customized its error handling behavior, what is displayed to the end user is the `errorSummary` of the first `ErrorCause` object that your external service returns.
```

Comment:
nit: I would change "...causing an error message to be displayed..." to just "...displaying an error message..."

Reply:
Made the change.